web/app/js/components/Navigation.jsx

```diff
@@ -356,7 +368,12 @@ class NavigationBase extends React.Component {
 
   render() {
     const { api, classes, selectedNamespace, ChildComponent, ...otherProps } = this.props;
-    const { namespaces, anchorEl, showNamespaceChangeDialog, newNamespace, mobileSidebarOpen } = this.state;
+    let { namespaces, namespaceFilter, namespaceFilterInput, anchorEl, showNamespaceChangeDialog, newNamespace, mobileSidebarOpen } = this.state;
```

Comment:
This is totally a style thing, but I like to keep everything under 80 chars so that it is readable. Looks like there are some jsfmt tools that work just like gofmt to do that for you.

Reply:
Good point - I use VSCode to softwrap my JS at 80 chars so didn't realize these lines were getting so long. Will make a ticket to update systematically.